specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2021-03-01-preview/mfe.json

```diff
@@ -0,0 +1,9458 @@
+          },
+          {
+            "in": "path",
+            "name": "endpointName",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          },
+          "204": {
+            "description": "Success"
+          },
+          "202": {
+            "description": "Success"
+          },
+          "200": {
+            "description": "Success"
+          }
+        },
+        "x-ms-examples": {
+          "Delete Batch Job Endpoint.": {
+            "$ref": "./examples/BatchJobEndpoint/delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/batchEndpoints/{endpointName}/jobs/{id}": {
+      "get": {
+        "tags": [
+          "BatchJob"
+        ],
+        "summary": "Gets a batch inference endpoint job by name.",
+        "operationId": "BatchJobEndpoints_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "endpointName",
+            "description": "Name of endpoint",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "id",
+            "description": "Identifier for the batch endpoint job.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/BatchJobResource"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Batch Job Endpoint.": {
+            "$ref": "./examples/BatchJobEndpoint/get.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/batchEndpoints/{endpointName}/deployments/{deploymentName}/jobs": {
```

Comment:
Same as above.  PUT should be at a single resource level, not at the resource type level

Reply:
removed batchJob, as it was cut from PuP today